Framework/DataHandling/src/LoadPSIMuonBin.cpp

```diff
@@ -225,6 +233,9 @@ void LoadPSIMuonBin::exec() {
   setProperty("TimeZero", absTimeZero);
   setProperty("TimeZeroList", correctedTimeZeroList);
 
+  // create time zero table
+  makeTimeZeroTable(correctedTimeZeroList);
```

Comment:
could we not use this instead https://github.com/mantidproject/mantid/pull/29687/files#diff-628b8f2fe46a8d269e7ceeeec935bf91R133

Reply:
The current implementation has createTimeZeroTable in LoadMuonStrategy as protected so it cannot be used publicly and LoadMuonStrategy is virtual. Also LoadPSIMuonBin does not use an instance of LoadMuonStrategy which is why a separate method was made. Do you think it's a good idea to change this implementation so LoadPSIMuon uses one of the single or multi-period strategies?

Reply:
it removes duplicate code, which could lead to bugs as we could only update one and forget about the other